Feature#2 main view

FranckZnt19 opened the pull request:
<!--- PR Title: What changed/implement in this PR -->
<!--- Be specific, don't write a generic title please --> 
 main view the sistem
## Description: Why did you change/implement that?
<!-- Describe your changes in detail -->
i created the main view of the system where the user check and register the hour only exist the view for index show and create
## Testing: Help me how can I test or look at the changes?
<!-- Please describe in detail how you tested your changes or use screenshots -->

Commit:
add attendace

add new scaffold for Attendace with columns numer_private, date and time

Commit:
update routes

declare except in routes edit and destroy in the attendace for not get this actions in the sistem

Commit:
update main view

update root route in the sistem for the oclok check of the users for be the principal view in the sistem

Commit:
validation

add validation of the column private_number for the form attendace

Commit:
add data time

add data and time like in the model of design for have at user notified of time

Commit:
hidden date and time

hidden inputs date and time in the form attendace for send in the backend automatic in the register the attendace

OswaldoPineda commented on reloj-checador/app/assets/stylesheets/scaffolds.scss:
```diff
@@ -1,65 +1,65 @@
-//   background-color: #fff;
-//   color: #333;
-//   margin: 33px; }
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px; }
```
try to use
```
body {
  background-color: #fff;
  color: #333;
  margin: 33px;
}
```
instead of
```
body {
  background-color: #fff;
  color: #333;
  margin: 33px; }
```
it's easier to read and maintain

OswaldoPineda commented on reloj-checador/app/helpers/attendaces_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module AttendacesHelper
+end
```
delete this file if it's not neccessary

OswaldoPineda commented on reloj-checador/config/database.yml:
```diff
@@ -26,8 +26,8 @@
   encoding: unicode
-  username: postgres
-  password: 123456 #<-- which you have defiend while installing postgresql
+  username: frania
+  password: 12345 #<-- which you have defiend while installing postgresql
```
it could be better to have a neutral username like `root`

FranckZnt19 replied:
[https://github.com/Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08/pull/16/commits/04f461c2e19281596bb07c5b184815dee4cc229d](add figaro gem for hidden params in the code)

OswaldoPineda commented on reloj-checador/spec/helpers/attendaces_helper_spec.rb:
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the AttendacesHelper. For example:
+#
+# describe AttendacesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
delete the comments if they are not necessary anymore

FranckZnt19 replied:
[https://github.com/Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08/pull/16/commits/cd57fa3b0805cd022c7fa33f759d0eb8ebed9474](removed the comments)

OswaldoPineda reviewed (commented):
and another thing, check the best practices on the commits 
https://chris.beams.io/posts/git-commit/
they look good but my small observation is about to start to use verbs like `Run`, `Fix`, `Update`, `Add`, `Remove`, etc

Commit:
resolved code comment

remove the code comment in this file

Commit:
remove personal data db

remove my personal data of the login in the db and add gem figaro for hidden this params of entry
